libs/ast_generic/AST_generic.ml

```diff
@@ -365,7 +365,7 @@ and id_info = {
    * a typed entity, which can be interpreted as a TypedMetavar in semgrep.
    * alt: have an explicity type_ field in entity.
    *)
-  id_type : type_ option ref;
+  id_type : type_ option ref; [@equal fun _a _b -> true]
```

Comment:
What is this for?

Reply:
I'm certain this is needed in the future, but I guess I'll delete for now